Remove mapping subcommand

aj-stein-nist opened the pull request:
# Committer Notes

 Remove mapping subcommand to align with 1.1.0 for #168. 

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/oscal-cli/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/oscal-cli/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- ~Have you included examples of how to use your new feature(s)?~
- ~Have you updated all website and readme documentation affected by the changes you made?~

david-waltermire-nist commented:
@aj-stein-nist To fully remove the command, you need to remove the line [here](https://github.com/usnistgov/oscal-cli/blob/865e379207f0f32c44f031f2131425c42f75b540/src/main/java/gov/nist/secauto/oscal/tools/cli/core/CLI.java#L75).

aj-stein-nist commented:
> @aj-stein-nist To fully remove the command, you need to remove the line [here](https://github.com/usnistgov/oscal-cli/blob/865e379207f0f32c44f031f2131425c42f75b540/src/main/java/gov/nist/secauto/oscal/tools/cli/core/CLI.java#L75).

It's ready now. `git rm -rf --cached` is foolish (not sure why I did that) but files were still on disk. I removed that and adjusted the test with all mappingcollection directory gone. It's good now and ready for re-review.
